tests/implementor/test_impl_spawner.py

```diff
@@ -68,12 +72,14 @@ def test_delete_spawner_configuration_not_allowed(
     def test_get_all_spawner_configuration_ids(self, token, spawner_configuration_data):
         config = SpawnerConfiguration.create(**spawner_configuration_data)
 
-        response = impl.component.get_all_spawner_configuration_ids({}, token)
+        response = impl.component.get_all_spawner_configuration_ids(
+            {"simulationId": None}, token
+        )
         (result, status) = response
```

Comment:
Wouldn't it be possible to change this like above already?
```suggestion
        (result, status) = impl.component.get_all_spawner_configuration_ids(
            {"simulationId": None}, token
        )
```

Reply:
Good idea 👍 I changed it!